app/controllers/backdate_controller.rb

```diff
@@ -3,6 +3,8 @@
 class BackdateController < ApplicationController
   def edit
     @edition = Edition.find_current(document: params[:document])
+    assert_edition_state(@edition, &:editable?)
+    assert_edition_state(@edition, &:first?)
```

Comment:
Is this intentionally 2 separate calls? I remember we touched on this briefly in conversation before and didn't conclude.

Reply:
Not really - I just found it more intuitive to write this way. It has the small benefit of automatically logging the check (see above comment).

Reply:
Discussed - we may prefer a one-liner depending on how much we repeat these checks.